test/fetch-test-deps.sh

```diff
@@ -40,6 +47,25 @@ while [[ $# -gt 0 ]]; do
 done
 
 case "$actionname" in
+	--get-deps)
```

Comment:
@ISSOtm There are other places we could put the Pillow-installing script; I hope this way makes sense.